WordPress/Classes/ViewRelated/Post/FeaturedImageViewController.m

```diff
@@ -39,21 +38,15 @@ - (void)viewDidLoad
 
     self.view.backgroundColor = [UIColor whiteColor];
 
-    [self setupToolbar];
+    UIBarButtonItem *rightFixedSpacer = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemFixedSpace target:nil action:nil];
+    rightFixedSpacer.width = -5.0f;
+    self.navigationItem.rightBarButtonItems = @[rightFixedSpacer, self.deleteButton];
```

Comment:
Maybe `[WPStyleGuide setRightBarButtonItemWithCorrectSpacing:forNavigationItem:]` instead of building the spacer here?